src/Nncase.Core/Utilities/ReplaceUtility.cs

```diff
@@ -91,4 +96,14 @@ public static Call ReplaceCallParams(Expr target, IReadOnlyList<Expr> oldParams,
     {
         return new Call(target, ImmutableArray.CreateRange(ReplaceItems(oldParams, pairs)));
     }
+
+    /// <summary>
+    /// replace the first params of call with expr.
+    /// </summary>
+    /// <param name="target">target.</param>
+    /// <param name="oldParams">oldParams.</param>
+    /// <param name="expr">expr.</param>
+    /// <returns>new Call.</returns>
+    public static Call ReplaceCallFirstParam(Expr target, IReadOnlyList<Expr> oldParams, Expr expr) =>
+        ReplaceCallParams(target, oldParams, (oldParams[0], expr));
```

Comment:
I think use `ReplaceCallParams(target, oldParams, (oldParams[0], expr))` is better clarify  than `ReplaceCallFirstParam(target,  oldParams, expr)`, There is no need to add this method.